api/swagger.yaml

```diff
@@ -1436,6 +1444,32 @@ paths:
         default:
           $ref: '#/responses/UnknownError'
 
+  /cisdockerbenchmarkresults/{id}:
+    get:
+      summary: 'Get CIS Docker Benchmark results by Application Resource'
+      parameters:
+        - $ref: '#/parameters/id'
+        - $ref: '#/parameters/cisDockerBenchmarkResultsSortKey'
+        - $ref: '#/parameters/sortDir'
+      responses:
+        '200':
+          description: 'Success'
+          schema:
+            type: 'object'
```

Comment:
maybe it's better to define a new object for the CIS docker benchmark result (that includes the total and the list of findings)?

Reply:
In the case of the other API endpoints where you can sort the result, the total filed is separated. So if we want to be consequent we should do the same here. WDYT?